src/components/DelegationActionBar/DelegationActionBar.module.css

```diff
@@ -24,8 +26,12 @@
   margin-bottom: 20px;
 }
 
-.accordionHeaderTitle{
+.actionBarHeaderTitle{
   font-weight: 500;
   margin-top: 2px;
   margin-bottom: 5px;
-}
\ No newline at end of file
+}
+
+.delegationActionBar{
+  margin-right: 10px;
```

Comment:
This is very much a nitpick, but I would probably put the extra space as a padding in the container that has the overflow and scroll bar, instead of a margin on every individual delegationActionBar. Feels more neat I think, since the scroll bar is the reason we're adding this space in the first place.

Reply:
Fixed